### PR TITLE
Post-release 0.32.0 updates

### PR DESCRIPTION
**Pull request by KirbyKatcher**
updated release instructions to have instructions with galasactl commands and removed some markdown issues
Shell script to run all of pre-release

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

File: releasePipeline/prerelease.md [hunk 1]
```diff
@@ -11,21 +12,29 @@ It may be beneficial to complete a pre-release before starting a vx.xx.x release
...
 5. Authenticate to the cicsk8s cluster using `cicsk8s sso`
 
+## Pre-release steps - Automated
+
+1. Ensure you have completed the [set up](#set-up) before continuing.
+2. Run [01-run-pre-release.sh](./01-run-pre-release.sh). If the process fails at any stage, you can continue by re-running the script that failed from the manual steps and finish using the [manual steps below](#pre-release-steps---manual).
+3. Run [25-check-artifacts-signed.sh](./25-check-artifacts-signed.sh). When prompted, choose the '`pre-release`' option.
+    - Each maven artifact should contain a file called com.auth0.jwt-<*VERSION*>.jar.asc. If the .asc files aren't present, debug and diagnose why the artifacts have not been signed.
+
+4. Send the [mvp image](https://development.galasa.dev/prerelease/maven-repo/mvp/dev/galasa/galasa-isolated-mvp) to Will Yates to perform the MEND scan to check for any vulnerabilities before moving onto the release process.
```
Comment (jadecarino) [inline releasePipeline/prerelease.md:22]: Minor point but let's add my name in here as well as Will, as if he is not available to do this, I can.

File: releasePipeline/prerelease.md [hunk 2]
```diff
@@ -11,21 +12,29 @@ It may be beneficial to complete a pre-release before starting a vx.xx.x release
...
     2. Delete all Tags that were created: [Tags](https://github.com/galasa-dev/helm/tags) - Next to a Tag, click the three dots, then 'Delete Tag' then 'Delete this Tag'.
 7. Run [20-build-all-code.sh](./20-build-all-code.sh). When prompted, choose the '`pre-release`' option.
-8. Run [25-check-artifacts-signed.sh](./25-check-artifacts-signed.sh). When prompted, choose the '`pre-release`' option. 
+8. Run [25-check-artifacts-signed.sh](./25-check-artifacts-signed.sh). When prompted, choose the '`pre-release`' option.
     - Each maven artifact should contain a file called com.auth0.jwt-<*VERSION*>.jar.asc. If the .asc files aren't present, debug and diagnose why the artifacts have not been signed.
 
-9. Send the [mvp image](https://development.galasa.dev/prerelease/maven-repo/mvp/dev/galasa/galasa-isolated-mvp) to Will Yates to perform the MEND scan to check for any vulnerabilities before moving onto the release process.
\ No newline at end of file
+9. Send the [mvp image](https://development.galasa.dev/prerelease/maven-repo/mvp/dev/galasa/galasa-isolated-mvp) to Will Yates to perform the MEND scan to check for any vulnerabilities before moving onto the release process.
```
Comment (jadecarino) [inline releasePipeline/prerelease.md:40]: Same here

File: releasePipeline/release.md [hunk 3]
```diff
@@ -113,9 +116,17 @@ The pipeline it kicks off is called `tag-galasa-*`. Takes about a minute to comp
...
+2. Upgrade the version of Galasa to the new development version in the galasa-prod Ecosystem CPS properties, using the galasa cli as show below. Upgrade the galasaecosystem.runtime.version, galasaecosystem.isolated.mvp.zip and galasaecosystem.isolated.full.zip properties.
+
+``` shell
+galasactl properties set --namespace galasaecosystem --name isolated.full.zip --value https://development.galasa.dev/main/maven-repo/isolated/dev/galasa/galasa-isolated/0.33.0/galasa-isolated-0.33.0.zip --bootstrap https://galasa-galasa-prod.cicsk8s.hursley.ibm.com/bootstrap
+galasactl properties set --namespace galasaecosystem --name isolated.mvp.zip --value https://development.galasa.dev/main/maven-repo/mvp/dev/galasa/galasa-isolated-mvp/0.33.0/galasa-isolated-mvp-0.33.0.zip --bootstrap https://galasa-galasa-prod.cicsk8s.hursley.ibm.com/bootstrap
+galasactl properties set --namespace galasaecosystem --name runtime.version  --value 0.33.0 --bootstrap https://galasa-galasa-prod.cicsk8s.hursley.ibm.com/bootstrap
```
Comment (jadecarino) [inline releasePipeline/release.md:124]: We should make it clear that value `0.33.0` in these examples should be replaced with the new Galasa development version. Alternatively could we script this with `--devVersion` being passed in?

File: releasePipeline/05-helm-charts.sh [hunk 4]
```diff
@@ -216,13 +216,35 @@ function check_helm_charts_released {
...
+        release_url="https://api.github.com/repos/galasa-dev/helm/releases/tags/$release_tag"
+        curl $url > $release_json_details -s
+
+        # var=$(sed -ne "s/^\\\$"url" *= *'\([^']*\)' *;.*/\1/p" $release_json_details)
+        # success $var
+        # tag_url="https://api.github.com/repos/galasa-dev/helm/tags/$release_tag"
+        # curl -X DELETE $url
+
+    done
+}
```
Comment (jadecarino) [inline releasePipeline/05-helm-charts.sh:236]: Does this commented out bit need to be finished?

**Review by jadecarino — CHANGES_REQUESTED** [review]
A couple of small changes but overall looks good, thanks for automating this process, will save us time!

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Review by eamansour — APPROVED** [review]

**Pull request merged**